Templates/unit_template.ps1

```diff
@@ -0,0 +1,84 @@
+<#
+.Synopsis
+   Template for creating DSC Resource Unit Tests
+.DESCRIPTION
+   To Use:
+     1. Copy to \Tests\Integration\ folder and rename MSFT_x<ResourceName>.tests.ps1
+     2. Customize TODO sections.
+
+.NOTES
+   Code in HEADER and FOOTER regions are standard and may be moved into DSCResource.Tools in
+   Future and therefore should not be altered if possible.
+#>
+
+
+# TODO: Customize these paramters...
+$DSCModuleName      = 'x<ModuleName>' # Example xNetworking
+$DSCResourceName    = 'MSFT_x<ResourceName>' # Example MSFT_xFirewall
+# /TODO
+
+#region HEADER
+if ( (-not (Test-Path -Path '.\DSCResource.Tests\')) -or `
+     (-not (Test-Path -Path '.\DSCResource.Tests\TestHelper.psm1')) )
+{
+    Throw @(
+        "The DSCResource.Tests folder could not be found in the root folder of the $DSCModuleName DSC Module to test."
+        "Please use Git to clone this repository to the root folder of the $DSCModuleName DSC Module that needs to be tested using the command:"
+        "git clone https://github.com/PowerShell/DscResource.Tests.git"
+    ) -join "`n"
+}
+Import-Module .\DSCResource.Tests\TestHelper.psm1 -Force
+$TestEnvironment = Initialize-TestEnvironment `
+    -DSCModuleName $DSCModuleName `
+    -DSCResourceName $DSCResourceName `
+    -TestType Unit 
+#endregion
+
+# TODO: Other Optional Init Code Goes Here...
+
+# Begin Testing
+try
+{
+
+    #region Pester Tests
+
+    InModuleScope $DSCResourceName {
```

Comment:
Done. I just copied the text from the Pester Wiki (as it is easy to understand I think).